FrameworkSystem/scripts/dirac-admin-proxy-upload.py

```diff
@@ -3,13 +3,23 @@
 # File :    dirac-admin-proxy-upload.py
```

Comment:
Do we want to keep this script? It seems redundant to me.

Reply:
IMHO this tool is sometimes useful.